content/security/authentication.md

```diff
@@ -947,6 +947,43 @@ export class JwtStrategy extends PassportStrategy(Strategy, 'myjwt')
 
 Then, you refer to this via a decorator like `@UseGuards(AuthGuard('myjwt'))`.
 
+
+#### OAuth Client
+
+If in our application we need a third-party authentication (services such as google, twitter, facebook, etc.), create a strategy using the OAuth2 protocol (Open Authorization), an open standard protocol that a secure API authorization like this way:
+
+```typescript
+import { PassportStrategy } from '@nestjs/passport';
+import { Strategy, VerifyCallback } from 'passport-google-oauth20';
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class GoogleStrategy extends PassportStrategy(Strategy, 'google') {
+    constructor() {
+        super({
+            clientID: 'CLIENT_ID',
+            clientSecret: 'CLIENT_SECRET',
+            callbackURL: 'CALLBACK_URL',
+            scope: ['email', 'profile'],
+        });
+    }
+    
+    async validate(accessToken: string, refreshToken: string, profile: any, done: VerifyCallback): Promise<any> {
```

Comment:
Why do you have `done` as a property when you don't use it and just return a promise instead?

Reply:
Can you commit me to the change? because I didn't quite understand what point you are referring to

Reply:
can you make me the change request? how did you do with the text?

Reply:
Ok, Done!